sk0-front/src/app/auth/auth.service.ts

```diff
@@ -57,8 +63,17 @@ export class AuthService {
         next: (user: LoginResponse) => {
           this._userName.next(user.name);
           this._userId.next(user.id);
-          this._isUserLoggedIn.next(true);
-          this.navCtrl.navigateForward("/home");
+          //TODO: better handling if token is null/empty?
+          if (user.token) {
+            this.localStore.saveToken(user.token);
```

Comment:
I totally agree with this statement. The uncomfortable truth is, that this is not really in parity with the other pr and whichever is getting merged first, the other needs to be rewritten.
Somewhat expected with bigger and longer PRs, it is not you it is us.

Reply:
It should be in a metareducer or similar, but definitely not the scope of this PR.